yarn.lock

```diff
@@ -11436,7 +11461,26 @@ eslint-plugin-es@^3.0.0:
     eslint-utils "^2.0.0"
     regexpp "^3.0.0"
 
-eslint-plugin-import@2.22.1, eslint-plugin-import@^2.22.1:
+eslint-plugin-import@2.25.4:
```

Comment:
Do you want to dedupe it with the next `eslint-plugin-import`entry?

Reply:
absolutely, thx for reminder!

Reply:
done